Makefile

```diff
@@ -31,82 +61,43 @@ sweep: ## destroy the whole architecture; USE ONLY FOR DEVELOPMENT ACCOUNTS
 			SNOWFLAKE_ENABLE_SWEEP=1 go test -timeout 300s -run ^TestSweepAll ./pkg/sdk -v; \
 			else echo "Aborting..."; \
 		fi;
-.PHONY: sweep
 
-lint-ci: ## run the fast go linters
-	./bin/reviewdog -conf .reviewdog.yml -reporter=github-pr-review -tee -fail-on-error=true
-.PHONY: lint-ci
+test: ## run unit and integration tests
+	go test -v -cover -timeout=30m ./...
 
-test-acceptance: ## runs all tests, including the acceptance tests which create and destroys real resources
-	SKIP_MANAGED_ACCOUNT_TEST=1 SKIP_EMAIL_INTEGRATION_TESTS=1 TF_ACC=1 go test -timeout 2000s -v $(COVERAGE_FLAGS) ./...
-.PHONY: test-acceptance
+test-acceptance: ## run acceptance tests
+	TF_ACC=1 go test -v -cover -timeout 30m
```

Comment:
This line is not running tests, check the GH action (content and time). You lack `./...` at the end.

Reply:
fixed